Cookie

Elaindil opened the pull request.

jerryweb committed:
Fixed Easy AI reinforcement values

Elaindil committed:
Changed easy values to be correct and disabled ai tagging

Elaindil committed:
Merge branch 'cookie' of https://github.com/jerryweb/Conquest-Enhanced-Mod into cookie

Elaindil committed:
Added smoke cover for tank AI

jerryweb committed:
Removed unused 20mm flak30 bone.

jerryweb committed:
Fixed salvage bug with sandbag_script. May not need external salvage script anymore

Elaindil committed:
Ai katy movement

jerryweb commented:
The Easy AI fix isn't their with the latest katy script. We need to add it back.

jerryweb committed:
Fixed Easy Difficulty Values and Player CP scaling with difficulty

changed player CP to scale downward when difficulty increases
changed normal ai cp attack multiplier to 4.0 for larger ai attacks

Elaindil committed:
Changed CP values on different difficulties, reworked existing divisions, hopefully fixed barrikady

Elaindil committed:
Merge branch 'cookie' of https://github.com/jerryweb/Conquest-Enhanced-Mod into cookie

Elaindil committed:
Fixed crash

Elaindil committed:
added pause after 30 minutes for AI that got removed somewhere

Elaindil committed:
Barrikady fix for counter attack on defense

Elaindil committed:
Changed no scavenge

Elaindil committed:
Changed wavedurations, narofominsk with testing wavesystem and change nebel research stage

Elaindil committed:
Changed wave system to all front attack

Elaindil committed:
LIVE PATCH

jerryweb committed:
Added 30% chance for random initial enemy wave spawn time. Increased number of possible maps from 3 to 5 on unlimited.

jerryweb committed:
added late arty noob division

jerryweb committed:
Fixed incorrect tags and extended spawn zone on side b

Elaindil committed:
Added a function to spawn stuff for infantry

Elaindil committed:
Merge branch 'cookie' of https://github.com/jerryweb/Conquest-Enhanced-Mod into cookie

Elaindil committed:
Added named tags to every vehicle

jerryweb committed:
Added Typhoon Wave Mode with a 50% chance of happening

Also reverted old wave duration values since typhoon mode overwrites all durations with 3 seconds as well as spawn cooldown times.

Elaindil committed:
changes to scripts

changed randomtimer condition to also randomize some scripts
removed seconds arty barrage for normal and easy
added randomizer for player defenses

Elaindil committed:
Merge branch 'cookie' of https://github.com/jerryweb/Conquest-Enhanced-Mod into cookie

Elaindil committed:
fixes

Elaindil committed:
Changes in description

added randomizer for player defenses
changed randomtimer condition to also randomize some scripts
removed seconds arty barrage for normal and easy
added all ai pole commands to all maps
added abandon the vehicle intercation file for ai
implemented katyscript changes to all maps

jerryweb committed:
Added Sandbag script and increased heroic and hard mac ai cp to 400

Elaindil committed:
Changed waypoints

Elaindil committed:
Merge branch 'cookie' of https://github.com/jerryweb/Conquest-Enhanced-Mod into cookie

jerryweb committed:
Added Typhon Wave system spawn scripts with correct waypoint values . Also Fixed Sandbag Script to use flak ring

flak ring effects 88 and 85 mm guns

jerryweb committed:
Fixed Sandbag texture issue and vehicle hull down bug

jerryweb committed:
Added sandbag ring build options for player. Added mgs to sandbag script. Fixed incorrect textures for bags

also changed sandbags to spawn over the 30 second

jerryweb committed:
Added the correct version of the missions (last one was prepatch)

jerryweb committed:
Updating to hotfix version

jerryweb committed:
Added tags to random blue men. Fixed at guns with 3 men

jerryweb committed:
Added variety for flak and mortar rings.

jerryweb committed:
Fixed Sandbag ring straight to have more space

jerryweb committed:
Changed Tank Attack Logic, Moved Hull Down panel control to middle bottom, fixed ai mg emplacements to no longer move by themselves if not spawned as wave

fixed mg emplacements from moving on map, changed tank attack logic so that light tanks always are effected. medium tanks aren't effected unless there are >= 3 flags and then they have a 50% chance to stop shoot move, else just stop and shoot. heavy tanks just stop and shoot
same with spgs
updated sandbag script to activate a bit slower
Separated spawn and wave code from bot file to bot_wave_system file
moved some waypoint on narvaminsk

jerryweb committed:
fix test values for first wave offset.

jerryweb committed:
fix the missing or broken scripts. Still need to check and fix all maps

jerryweb committed:
Fixed Typhoon wave spawns for all maps and deleted old ai spawn move system. Added typhoon wave mode toggle

mechanic for use to be able to dynamically toggle off and on typhoon wave mode ingame
can configure it to be able to change based on conditions we want in the bot file or just a random value
When toggled on, there is a 50% of toggling the typhoon wave mode on or off. Interval between possible wave toggling is set to 300 seconds or 5 mins

jerryweb committed:
Expanded map size for player spawn camping on bykovo and ruetzen

jerryweb committed:
Updated hillls, ruetzen and bykovo

jerryweb committed:
Adding large file storage support for the project

jerryweb committed:
Added extra map space to all maps. Still need to fix map borders and spawn locations.

jerryweb committed:
Fixed a couple maps that did not have extended spawn locations

jerryweb committed:
Fixed mg script

jerryweb committed:
Pushing latest steam release version version

jerryweb committed:
fixed mg script again

jerryweb committed:
Added paratrooper missions folder

jerryweb committed:
Added ai_poles to every map

jerryweb committed:
Fixed incorrect cp values for bots
